### ISSUE TITLE
Model Testing 

### ISSUE THREAD

**Issue by saja1994** [MuhammedBuyukkinaci/TensorFlow-Sentiment-Analysis-on-Amazon-Reviews-Data#3]
firstly thank you for your effort. Please, I have a question. I'm a little bit confused about why you are using the testing data for the validation process? Because as far as I know there should be training, validation, and testing. But I don't see that in your implementation! 

Also, I want to ask if there is a way to visualize the attention weights in 11_Attention_GRU file? 

Thanks

**Comment by MuhammedBuyukkinaci** [comment]
The validation strategy may change. You might be right. Sorry for the late answer.

**Comment by saja1994** [comment]
So can we consider your validation as a testing process in your code?
Thanks

**Comment by MuhammedBuyukkinaci** [comment]
I think the best validation strategy is to split as train,test and validation. We should train our models on train set, validate it on validation set and score it on test set. Maybe I should update my code. Thanks for your beneficial feedbacks.

**Comment by Jon-Zbw** [comment]
If I want to do sound emotion recognition, I could extract the mfcc, which py files I can refer to you？
